src/hydra/hydraClient.js

```diff
@@ -6,12 +6,12 @@ import {
   GET_MANY_REFERENCE,
```

Comment:
Maybe can we rename this file `dataProvider.js`?

Reply:
This hasn't been done for React Admin built-in clients. I don't think it makes sense. (a restClient is a kind of data provider)